lotto/client/js/LottoInput.js

```diff
@@ -0,0 +1,34 @@
+import {EVENT_TYPE, LOTTO} from "./constants.js";
+
+export function LottoInput({onAdd}) {
```

Comment:
한 파일에서 여러 컴포넌트/함수들을 `export` 하는 것이 아니라면 `export default` 구문을 사용하는 것이 좋습니다.

```suggestion
function LottoInput({onAdd}) {
// ...
export default LottoInput;
```